### PR TITLE
Fix research links redirects

### PR DESCRIPTION
**Pull request by SYangster**
### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: research/auto-fed-rl/index.html
Comment (YuanTingHsieh) [inline research/auto-fed-rl/index.html:1]: so we don't need these index.html now?

Comment (SYangster) [reply research/auto-fed-rl/index.html:1]: Right, now that we are using https://nvidia.github.io/NVFlare/ for our website, these index.html redirects aren't needed anymore

Comment (YuanTingHsieh) [reply research/auto-fed-rl/index.html:1]: got it, sounds good.

**Review by YuanTingHsieh — APPROVED** [review]
LGTM

**Pull request merged**